src/Hungary/Mews.Fiscalizations.Hungary.Tests/InvoiceTests.cs

```diff
@@ -285,9 +284,8 @@ private ModificationInvoice CreateModificationInvoice(InvoiceNumber originalDocu
                 )
             };
 
-            var number = new Random().Next(1, 10000);
             return new ModificationInvoice(
-                number: InvoiceNumber.Create($"INVOICE-{number}-REBATE").Success.Get(),
+                number: InvoiceNumber.Create($"REBATE-{Guid.NewGuid()}").Success.Get(),
```

Comment:
Just wondering if it wasn't important to have the `INVOICE-` prefix to this to match the original invoice number?

Reply:
this is the new invoice number
below is the originalDocumentNumber (line 300/298)